admin/class-meta-storage.php

```diff
@@ -58,69 +58,47 @@ public function install() {
 	}
 
 	/**
-	 * Removes the record for given post_id.
-	 *
-	 * @param int $object_id The post_id to remove the record for.
-	 *
-	 * @return int|false The number of rows updated, or false on error.
-	 */
-	public function cleanup( $object_id ) {
-		$deleted = $this->database_proxy->delete(
-			array( 'object_id' => $object_id ),
-			array( '%d' )
-		);
-
-		if ( $deleted === false ) {
-			WPSEO_Meta_Table_Accessible::set_inaccessible();
-		}
-
-		return $deleted;
-	}
-
-	/**
 	 * Saves the link count to the database.
 	 *
 	 * @param int   $meta_id   The id to save the link count for.
 	 * @param array $meta_data The total amount of links.
 	 */
 	public function save_meta_data( $meta_id, array $meta_data ) {
-		$inserted = $this->database_proxy->insert(
-			array_merge(
-				array( 'object_id' => $meta_id ),
-				$meta_data
-			),
-			array( '%d', '%d' )
+		$where = array( 'object_id' => $meta_id );
+
+		$saved = $this->database_proxy->upsert(
+			array_merge( $where, $meta_data ),
+			$where
 		);
 
-		if ( $inserted === false ) {
+		if ( $saved === false ) {
 			WPSEO_Meta_Table_Accessible::set_inaccessible();
 		}
 	}
 
 	/**
-	 * Updates the incoming link counts
+	 * Updates the incoming link count
 	 *
-	 * @param WPSEO_Link_Storage $storage The link storage object.
+	 * @param array              $post_ids The posts to update the incoming link count for.
+	 * @param WPSEO_Link_Storage $storage  The link storage object.
 	 */
-	public function update_incoming_link_counts( WPSEO_Link_Storage $storage ) {
+	public function update_incoming_link_count( array $post_ids, WPSEO_Link_Storage $storage ) {
```

Comment:
The method has a different name and this was introduced in 5.0.0; the change that this is implemented is virtually 0.